.github/workflows/reusable-commit-signature-checker.yml

```
@@ -0,0 +1,34 @@
+---
+on:
+  workflow_call:
+
+permissions: {}
+
+jobs:
+  commit-signature-checker:
+    name: Commit Signature Checker
+    runs-on: ubuntu-latest
+    steps:
```

```suggestion
    runs-on: ubuntu-latest
    permissions:
      contents: read
      pull-requests: wriite
    steps:
```
I think it will need these permissions:

- `contents` to read the repository contents
- `pull-requests` to create the comment
  - https://github.com/ministryofjustice/analytical-platform-airflow-github-actions/blob/main/.github/workflows/shared-scan-container.yml#L20